mathjs/mathjs.d.ts

```diff
@@ -1334,6 +1334,10 @@ declare namespace mathjs {
 	export interface MathNode {
                 isNode: boolean;
                 isSymbolNode: boolean;
+                isConstantNode: boolean;
```

Comment:
i do not see any reference to `isConstantNod` or `isOperatorNode` in the documentation. can you provide more details?